resources/zib/zib-Pregnancy.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-Pregnancy" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-Pregnancy" />
+  <name value="ZibPregnancy" />
+  <title value="zib Pregnancy" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="Pregnancy or gravidity is the condition in which a woman carries a fertilized egg, which grows into an embryo and then a fetus. A pregnancy of between 37 and 42 weeks is considered normal. “Term pregnancy” is 37 to 42 weeks. The term gravidity is also used for the number of times a woman has been pregnant. This and parity - the number of times a woman has given birth - provides insight into the patient’s obstetric history." />
+  <purpose value="This Condition resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) Pregnancy-v4.0(2020EN)](https://zibs.nl/wiki/Pregnancy-v4.0(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-pregnancy-v4.0-2020EN" />
+    <uri value="https://zibs.nl/wiki/Pregnancy-v4.0(2020EN)" />
+    <name value="zib Pregnancy-v4.0(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Condition" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Condition" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Condition">
+      <path value="Condition" />
+      <short value="Pregnancy" />
+      <alias value="Zwangerschap" />
+      <mapping>
+        <identity value="zib-pregnancy-v4.0-2020EN" />
+        <map value="NL-CM:7.14.1" />
+        <comment value="Pregnancy" />
+      </mapping>
+    </element>
+    <element id="Condition.note">
+      <path value="Condition.note" />
+      <max value="1" />
```

Comment:
Ik snap deze keuze in het kader van de zib, maar in de praktijk kan het weleens voorkomen dat er meerdere soorten notities zijn. Door de max cardinaliteit te beperken, kunnen implementers dit element niet meer voor andere notities gebruiken. Dit beperkt de mogelijkheden voor hergebruik van het profiel. Aan de andere kant is er in dit element geen mogelijkheid om de "zib" notitie te differentieren van andere notities. Dus voor nu maar zo laten. Ik zag dat dit in de zib 2017 ook zo gemodelleerd is en er zijn tot nu toe blijkbaar nog geen klachten over gekomen.